aries_cloudagent/protocols/issue_credential/v2_0/formats/ld_proof/handler.py

```diff
@@ -518,7 +531,8 @@ async def store_credential(
             cred_value=credential.serialize(),
             given_id=credential.id,
             record_id=cred_id,
-            cred_tags=None,  # Tags should be derived from credential values
+            # cred_tags=None,  # Tags should be derived from credential values
+            cred_tags=cred_tags,
```

Comment:
This change is giving me errors when trying to issue a credential. `expanded_type is an array which is not supported by Indy storage. A tag must be of type string.

This also makes it hard to search I guess, because you can only search for the whole value. Maybe we can take the same approach as with the non-expanded types and store it per value. e.g.:

```
tags[f"type:xpnd:{expanded_type_val}"] = "1"
```

Then when searching we can search per type. What do you think?

Reply:
I was saving it as tags[expanded_type_val] = "1", but I will adopt this approach.

Reply:
Saving it as: ` tags[f"xpnd:type:{expanded_type_val}"] = "1"`, with `type:xpnd` I was running into an issue where upon storing the credential, the expanded type was being saved into `types` and not `cred_tags`.